.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    - id: trailing-whitespace
+    - id: check-added-large-files
+    - id: mixed-line-ending
+      args: ['--fix=lf']
+
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v5.8.0
+    hooks:
+    -   id: isort
+        args: ['--multi-line=3', '--trailing-comma', '--force-grid-wrap=0', '--use-parentheses', '--line-width=88']
```

Comment:
Calling this out because the settings look strange at first.

These are the recommended settings for interop with black (which also may adjust imports)

Reply:
Makes sense. Thanks for pointing this out! Just going to plop [the reference doc](https://pycqa.github.io/isort/docs/configuration/profiles/) here for my own reference.